lexers/l/lasso.go

```diff
@@ -0,0 +1,26 @@
+package l
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// Lasso lexer.
+var Lasso = internal.Register(MustNewLexer(
+	&Config{
+		Name:    "Lasso",
+		Aliases: []string{"lasso", "lassoscript"},
+		Filenames: []string{
+			"*.lasso",
+			"*.lasso[89]",
+			// alias filesnames from pygments
```

Comment:
There's a dedicated attribute to inform aliases for filenames -> `AliasFilenames`